test/scenarios.test.ts

```diff
@@ -56,7 +56,7 @@ describe("scenarios", () => {
         resourceGroup: resourceGroup,
         templateFile: `${basePath}/main.bicep`,
         parametersFile: `${basePath}/main.bicepparam`
-      });
+      }, async (heading: string, body: string[][]) => {});
```

Comment:
@anthony-c-martin Let me know if there's a better way to mock this, I spent some time to play with jest but can't get jest mock stuff working...